packages/driver/cypress/e2e/e2e/origin/cookie_misc.cy.ts

```diff
@@ -0,0 +1,208 @@
+// FIXME: currently cookies aren't cleared properly in headless mode with webkit between tests, as the below tests (excluding cy.origin) pass headfully locally.
+describe('misc cookie tests', { browser: '!webkit' }, () => {
```

Comment:
Any reason we can add these tests to the existing e2e/cookie_behavior spec?

Reply:
main reason is just how large the spec already is. I almost think we should rename it, just not sold on something yet

Reply:
I guess I'd rather have one place to look for tests related to a topic so it's easier to see if we have coverage rather than searching multiple files 🤷🏻‍♀️

Reply:
I agree, but considering how flaky that spec currently is, I don't want real issues to get lost in the noise. Not a great reason, but I also think the two specs are accomplishing two different things. This one is more for expiry and duplication bug fixes, and the cookie behavior tests are mostly checking attachment for xhr/fetch requests. 

Would it be better to break the `misc` spec into two specs with the issues they are solving to show they serve a purpose for a regression test?

Reply:
maybe group the various cookie tests under a cookie directory?

Reply:
I'm gonna have a follow up to this PR that is going to address types, as well as refactoring some of these tests and addressing some of the cookie behavior flake